### DISCUSSION TITLE
Request for Custom Game Point per set

### DISCUSSION THREAD

**Discussion by yeremiaaryo** [matco/badminton#28] [Ideas]
Hi, I was wondering if we can customize the point per set (not strictly 21).
As a hobbyist, usually we play 25-30 points per set
Thank you!

**Comment by matco** [comment]
Hello!

It's already possible to configure the number of points of a set in the application settings, using in the Garmin Connect app on your smartphone or Garmin Express on your computer.

For detailed instructions, take a look at the following links:
- https://support.garmin.com/en-US/?faq=SPo0TFvhQO04O36Y5TYRh5 for the Garmin Connect app
- https://support.garmin.com/en-US/?faq=6Vl7EbW3qA42l63ZIhJet5 for Garmin Express
